spec/fixtures/fixtures.yml

```diff
@@ -6,31 +6,31 @@ apache-with-readme-notice:
   hash: ab3901051663cb8ee5dea9ebdff406ad136910e3
 bsd-3-lists:
   key: bsd-3-clause
-  matcher: 
-  hash: 
+  matcher:
```

Comment:
Was this file manually edited? Though it's not bad to get rid of superfluous spaces in general, it is if the next update with `bundle exec dump-detect-json-fixture` just adds them back in. 😄

Reply:
I did manually add it here and my editor settings clipped the trailing spaces. I'll use the script to regenerate this instead.